spacy/pipeline/morphologizer.pyx

```diff
@@ -101,8 +97,8 @@ class Morphologizer(Tagger):
         model: Model,
         name: str = "morphologizer",
         *,
-        overwrite: bool = BACKWARD_OVERWRITE,
-        extend: bool = BACKWARD_EXTEND,
+        overwrite: bool = False,
+        extend: bool = True,
```

Comment:
I don't understand the motivation for changing this from the current default?

```suggestion
        extend: bool = False,
```

Reply:
I don't really have strong thoughts about it, but it seemed reasonable (like a mode I would be likely to use) because it only adds. It's fine to make it false as well.

Reply:
Maybe there could be good reasons to have it be true?

Reply:
I think it's okay to leave it as proposed.

Reply:
Wait, this was changed, so it's not as proposed.

Reply:
But I think I like `False` here so it doesn't change from v3.

Reply:
To be clear it is currently `False` and not what I argued for above. But I'm OK with either way - while I do think the argument above makes sense, I don't feel very strongly about it, and it's not like it's that hard to change.

Reply:
Let's keep `False` for now then, sorry for the mixed-up comments above.